common/conf.py

```diff
@@ -149,6 +149,7 @@ def get_previous_version(majornumber, minornumber=0):
 downloads_root = 'https://downloads.openmicroscopy.org'
 help_root = 'https://help.openmicroscopy.org'
 docs_root = 'https://docs.openmicroscopy.org'
+rtd_docs_root = 'https://ome-contributing.readthedocs.io/en/latest/'
```

Comment:
That does not feel like the root for all readthedocs content. Did you mean `contributing_docs_root`?

Reply:
yep. sorted